config/v1/0000_10_config-operator_01_imagecontentpolicy.crd.yaml

```diff
@@ -45,20 +45,20 @@ spec:
                     required:
                       - source
                     properties:
-                      allowMirrorByTags:
```

Comment:
Breaking a v1 API stands out. It is very rare that this proper.  Perhaps an accident in this API PR?

Reply:
This API is not effective, there's no underlying support for this API. This API was planned to support the epic https://issues.redhat.com/browse/OCPNODE-521. But people think have reservations about it so we did not continue on development for this API.

Reply:
This shipped in 4.10 and the validation regex below also shipped in 4.10.  If we change this validation, I can create manifests that validate successfully against 4.10, but are rejected in 4.11.  Try containing your changes to the new API you want while leaving the existing types alone.

Reply:
Fixed. the new CRD reuses the naming of repositoryDigestMirror of the struct in this non-effective API, I update the PR to not reuse it. So the validation will not break that v1 API.